### ISSUE TITLE
Add Enable/Disable Config/Constants & Coefficient Feature to Edit Part Types

### ISSUE THREAD

**Issue by joannekoch** [WHOIGit/ooicgsn-roundabout#180]
Make the Config/Constants & Coefficient Feature configurable in the Part Type definition.

Rational: Constants/configs & coefficients are only implemented for Instruments and Electrical Part Types and Assemblies and Inventory built from them.  If a user wishes to define a sub Part Type of Instruments and wants this feature for those sub Part Types or if a user defines a new Part Type and wants this feature for the new Part Type, this enhancement will allow it.

**Comment by ethanandrews** [comment]
@rhmunier @cahearn @spetillo  This feature is now up on rdb-testing for... testing:

https://rdb-testing.whoi.edu/parts/part_type/

**Comment by rhmunier** [comment]
maybe not something for 1.5.2 but could we add a way to collapse the custom fields section?  Adding the CCC sections back puts them at the bottom and with the number of custom fields for some part types leads to a lot of extra scrolling.  No functional issues that I have found though.

**Comment by spetillo** [comment]
I submitted @rhmunier 's comment above as #233 .

This feature works well! On Staging, I tried enabling/disabling all Part Types, including Instruments that already had CCCs associated with Inventory. Disabling hid existing CCCs and associated tabs/functionality/menu iitems, and Enabling showed them again, as expected.

I added a follow-on card #234 to later address/hide the Disabled CCC Events that still show up in CCC Search results.
